src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl

```diff
@@ -70,8 +70,7 @@ task modify_file_gcs {
     Boolean ready
     String file_path_raw
     command {
-        echo "override" > tmp
-        gsutil cp tmp ${file_path_raw}
+        echo "override" | gsutil cp - ${file_path_raw}
```

Comment:
Actually I think I'd prefer this stay as it is and the tmpdir naming should be made collision-resistant over in Cromwell.